File: server/recorder/src/main/java/org/rliz/cfm/RecorderApplication.java
package org.rliz.cfm;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.cloud.netflix.feign.EnableFeignClients;
import org.springframework.context.annotation.ComponentScan;

/**
 * Main class to start the application.
 */
@EnableAutoConfiguration
@ComponentScan("org.rliz.cfm")
@EnableFeignClients
public class RecorderApplication {

    public static void main(String[] args) throws Exception {
        SpringApplication.run(RecorderApplication.class, args);
    }
}

File: server/recorder/src/main/kotlin/org/rliz/cfm/recorder/RecorderApplication.kt
package org.rliz.cfm.recorder

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.context.annotation.ComponentScan
import org.springframework.scheduling.annotation.EnableScheduling

@EnableAutoConfiguration
@ComponentScan("org.rliz.cfm")
@EnableScheduling
class RecorderApplication

fun main(args: Array<String>) {
    SpringApplication.run(RecorderApplication::class.java, *args)
}
